source/Core/Src/settingsGUI.cpp

```diff
@@ -616,7 +616,7 @@ static void displayAnimationLoop(void) { OLED::drawCheckbox(getSettingValue(Sett
 
 static void displayBrightnessLevel(void) {
 
-  OLED::printNumber((getSettingValue(SettingsOptions::OLEDBrightness) / 11 + 1), 2, FontStyle::LARGE);
+  OLED::printNumber((getSettingValue(SettingsOptions::OLEDBrightness) / 25 + 1), 2, FontStyle::LARGE);
```

Comment:
I think we should extract this magic 25 as a contact of like BRUGHTNESS_STEP ?

Reply:
Does this meet your request?

Reply:
Does this match your request?

Reply:
Does this match your request? 😊